storage/json-api/src/main/resources/client_secrets.json

```diff
@@ -0,0 +1 @@
+{"installed":{"client_id":"122681785480-35mkukq7o3hh55i2uuftv0hp168hauet.apps.googleusercontent.com","project_id":"gcs-code-samples","auth_uri":"https://accounts.google.com/o/oauth2/auth","token_uri":"https://accounts.google.com/o/oauth2/token","auth_provider_x509_cert_url":"https://www.googleapis.com/oauth2/v1/certs","client_secret":"hf-15vXoQQY8OonIK7qdrR0L","redirect_uris":["urn:ietf:wg:oauth:2.0:oob","http://localhost"]}}
```

Comment:
I'm confused by this. Applications ship client IDs client secrets. For example, our own command-line client, gsutil, ships with its own client secret. Why should a sample app not be the same?